hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/TestOMRequestUtils.java

```diff
@@ -439,6 +441,21 @@ public static void addBucketToDB(String volumeName, String bucketName,
         new CacheValue<>(Optional.of(omBucketInfo), 1L));
   }
 
+  public static void addBucketToDB(String volumeName, String bucketName,
```

Comment:
same here. Can we avoid copying the method?

Reply:
Done